### ISSUE TITLE
<dateMadeAvailable> not making it into returned metadata

### ISSUE THREAD

**Issue by axemanyo10** [lulibrary/puree#3]
Hi there,

I'm using v0.17 and no matter which PURE UUIDs I try, "available" always comes back empty, even when the XML from PURE seems to have `<dateMadeAvailable>` fully populated for the dataset in question. I've had a look through the puree code and in /lib/puree/dataset.rb I'm wondering if the extract_available method should have a "/" before "dateMadeAvailable", as in:

```
def extract_available
      temporal_start_date '/dateMadeAvailable'
end
```

The reason I think that is because without the "/", following the chain of method calls, the resulting xpath query from this method looks like it would be:

`/GetDataSetsResponse/result/contentdateMadeAvailable`

when surely it ought to be

`/GetDataSetsResponse/result/content/dateMadeAvailable` 

Alternatively, the missing "/" could go in the temporal_start_date method. Apologies in advance if I've got this wrong or I'm missing something obvious!

Cheers,
Fergus :)

**Comment by aalbinclark** [comment]
Good catch Fergus!

Fixed in 0.17.1

**Comment by axemanyo10** [comment]
Thanks very much! :+1:
